src/store/actions/activities.js

```diff
@@ -1,9 +1,46 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+
+const dataSanitize = (data) => {
+  if(data){
+    const { 
+  title_primary,
+  title_complementary,
+  description,
+  how_we_work_cards} = data;
+  const activitiesFilter = how_we_work_cards.map(
+    ({
+      _id,
+      Title,
+      description,
+      image_main
+    }) => {
+      return {
+        _id,
+        Title,
+        description,
+        image_main
```

Comment:
I think we only need url from `image_main`. I think other properties will cause extra network load. 
![Capture](https://user-images.githubusercontent.com/8650435/84525635-0176a200-aced-11ea-915e-274989dbd53b.PNG)
.